avaframe/ana5Utils/DFAPathGenerationCfg.ini

```diff
@@ -43,3 +43,6 @@ dsMin = 20
 
 # True to get the path from particles, False to get the path from fields (FT and FM)
 pathFromPart = True
+
+# parameters for scipy.interpolate splprep
+uInterval = 1000
```

Comment:
Didn't we say to get rid of this? And it is not used anymore, right?

Reply:
is not used anymore